pegasus/sites.v3/code.org/public/matching-gifts.haml

```diff
@@ -0,0 +1,31 @@
+---
+title: Matching Gifts
+theme: responsive_full_width
+---
+
+:scss
+  :root {
+    font-size: 14px;
+  }
+
+  #dd-container {
+    margin-top: 3em;
+  }
+
+:javascript
+  var DDCONF = { API_KEY: "4RoxGGsUoWBRMUfj" };
```

Comment:
I need more context here –– where is that API key coming from, and what does it do? I do think it best to avoid having this in our codebase if possible, but there may be other security measures I'm not aware of

Reply:
@davidsbailey has more context about putting JS in Haml, but I would also say to avoid that if possible. I'd be more inclined to add a line that ignores the linting error (https://eslint.org/docs/latest/use/configure/rules#disabling-rules)

Reply:
As a first step, I would suggest passing this secret down from the server via a secret, like we do for example with `CDO.mapbox_access_token`. I know it is a slippery slope, but I'm somewhat hesitant to make you extract a new js file for just one line of javascript. @kelbyhawn do you want to take a crack at copying the approach used for the mapbox token, or would you be happier having someone on our side take care of it?

Reply:
@davidsbailey sorry about this! I'll make a note to bring up API stuff in Slack 👍 

If things weren't so busy I'd try to figure it out, but I need to focus on finishing up other tasks before the curriculum launch. If one of y'all could take care of this I'd appreciate it!

Reply:
@megcrenshaw Dave helped set up the secret API — can you do a final review pass on this when you get the chance?